### PR TITLE
changes for #1726

### LINKED ISSUE #1726: Percentage of confirmed, active, recovered and decreased cases are showing same in the particular district in the map.
**Describe the bug**
Percentage of the confirmed, active, recovered and decreased cases are same in a particular district on the map but it's should be different as it is calaculated on the basis of the percentage of recovered, confirmed and decreased in a particular states accordingly.

**To Reproduce**
Show proper percentage of confirmed, active, recovered and decreased cases .

1. Go to '[ click to site](covid19india.org)'
2. Click on 'map'
3. Select particular state map.
4. See the percentage of confirmed, active, recovered and decreased cases for a particular district.

**Screenshots**
Confirm
active
recovered
decreased
See this as reference it's example of one of the district of a particular state.

### PR DESCRIPTION
**Pull request by ashishdawale20**
**Description of PR**

changes to fix the percentage on hover for different views. 
Changes to fix #1726 

**Screenshots**
After fix 
varanasi1
varanasi2
varanasi3
varanasi4

### CODE REVIEW

**Comment by bee-rickey** [comment]
Can you link the https://github.com/covid19india/covid19india-react/issues/1726 in your PR so that it gets auto closed?

**Comment by ashishdawale20** [comment]
@bee-rickey - Done

File: src/components/choropleth.js [hunk 1]
```diff
@@ -149,6 +149,7 @@ function ChoroplethMap({
         .on('click', handleClick)
         .attr('pointer-events', 'none');
 
+      regionSelection.select('title').remove();
```
Comment (jeremyphilemon) [inline src/components/choropleth.js:152]: Hey, could you check your PR again? The percentage fix doesn't seem to be here.

Let me know once you've resolved that :)

**Review by jeremyphilemon — CHANGES_REQUESTED** [review]

File: src/components/choropleth.js (hunk 1 above)
Comment (ashishdawale20) [reply src/components/choropleth.js:152]: @jeremyphilemon 
The percentage calculation was never a problem. The calculation is happening correctly. 
Every time user clicks on the different graph option the  "title" is getting appended with old titles. 

image

As the topmost title is the oldest one, on hover it shows that. 

So, the fix to this issue is to remove the old title and add a new one.
You can check here: https://covid19india-react-git-fork-ashishdawale20-master.covid19india.now.sh/

**Comment by shuklaayush** [comment]
Hey @ashishdawale20, sorry I didn't know that you had made a PR for this. I fixed it in #1712

**Pull request closed without merging**